pulsar-broker/src/main/java/org/apache/pulsar/broker/namespace/NamespaceService.java

```diff
@@ -880,12 +881,22 @@ void splitAndOwnBundleOnceAndRetry(NamespaceBundle bundle,
                                     for (NamespaceBundle sBundle : splittedBundles.getRight()) {
                                         checkNotNull(ownershipCache.tryAcquiringOwnership(sBundle));
                                     }
+                                    updateNamespaceBundlesForPolicies(nsname, splittedBundles.getLeft())
```

Comment:
What about update `updateNamespaceBundlesForPolicies` success but `updateNamespaceBundles` fail?

Reply:
After the split, several zookeeper paths are updated or created:
1. Each bundle corresponds to the path of the owner broker;
2. Update the local policies path;

Do you want to update the operations of these paths into atomic operations?

Reply:
It will retry!
<img width="1265" alt="image" src="https://user-images.githubusercontent.com/19296967/193182665-d78af75c-a22f-4e6b-bfc4-22d4ca57c3af.png">

Reply:
The maximum number of retry is 7 ：
<img width="885" alt="image" src="https://user-images.githubusercontent.com/19296967/193182877-79600acf-7776-4264-95dc-593ad6a4e212.png">

Reply:
Modify the execution order of the two methods to ensure that updateNamespaceBundlesForPolicies is executed after updateNamespaceBundles is completed.
@mattisonchao PTAL,thanks!